### PR TITLE
README with installation instructions and routes

### LINKED ISSUE #4: create a README
**User Story:**
As a developer, I need a README.md, so that I know the API and how to run and test the code

**Assumptions:**
* Can assume that users do not need instructions for installation of prerequisites such as `vagrant`, `docker`, etc

**Acceptance Criteria:**
AC1) A README.md is committed to the repository with an overview of how to use the code

### PR DESCRIPTION
**Pull request by biegel**
Addresses #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+None
+
+#### Body
+None
+
+#### Returns
+| HTTP code | Body | Description | 
+| --------- | ---- | ----------- |
+| `200`     | Wishlist items | All the existing items on the wishlist |
+| `400`     | Error JSON object | The wishlist `:Id` does not exist or not given |
```
Comment (ZiyiHuang96) [inline README.md]: The HTTP status code should be 404 if the wishlist `id` does not exist @biegel

File: README.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+None
+
+#### Body
+None
+
+#### Returns
+| HTTP code | Body | Description | 
+| --------- | ---- | ----------- |
+| `200`     | Wishlists JSON object | The wishlist name has been updated successfully |
+| `400`     | Error JSON object | Wishlist `:id` was not found or was missing |
```
Comment (ZiyiHuang96) [inline README.md]: The HTTP status code should be 404 if the wishlist id does not exist @biegel

File: README.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+None
+
+#### Body
+None
+
+#### Returns
+| HTTP code | Body | Description | 
+| --------- | ---- | ----------- |
+| `200`     | Wishlists JSON object | The JSON object of the wishlist deleted |
+| `400`     | Error JSON object | Wishlist `:id` was not found or was missing |
```
Comment (ZiyiHuang96) [inline README.md]: The HTTP status code should be 404 if the wishlist id does not exist @biegel

File: README.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+None
+
+#### Body
+None
+
+#### Returns
+| HTTP code | Body | Description | 
+| --------- | ---- | ----------- |
+| `200`     | Wishlist item JSON object | The JSON object of the wishlist item requests |
+| `400`     | Error JSON object | `:WishlistId` or `:ItemId` were not found or were missing |
```
Comment (ZiyiHuang96) [inline README.md]: The HTTP status code should be 404 if the `:WishlistId` or `:ItemId` does not exist @biegel

File: README.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+{
+   ...
+}
+```
+
+#### Returns
+| HTTP code | Body | Description | 
+| --------- | ---- | ----------- |
+| `200`     | Wishlist item JSON object | The JSON object of the wishlist item with updated data |
+| `400`     | Error JSON object | `:WishlistId` or `:ItemId` were not found or were missing |
```
Comment (ZiyiHuang96) [inline README.md]: The HTTP status code should be 404 if the `:WishlistId` or `:ItemId` does not exist or missing @biegel

File: README.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+None
+
+#### Body
+None
+
+#### Returns
+| HTTP code | Body | Description | 
+| --------- | ---- | ----------- |
+| `200`     | Wishlist item JSON object | The JSON object of the wishlist item deleted |
+| `400`     | Error JSON object | `:WishlistId` or `:ItemId` were not found or were missing |
```
Comment (ZiyiHuang96) [inline README.md]: The HTTP status code should be 404 if the `:WishlistId` or `:ItemId` does not exist or missing @biegel

File: README.md [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+| `500`     | Error JSON object | Server error |
```
Comment (ZiyiHuang96) [inline README.md:195]: I think we also need an endpoint for listing all Items in one wishlist? [Link](https://github.com/NYU-DevOps-Squad-Wishlists/wishlists/issues/10) @biegel

**Review by ZiyiHuang96 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (biegel) [reply README.md]: good call!  will update

File: README.md (hunk 4 above)
Comment (biegel) [reply README.md]: 👍

File: README.md (hunk 7 above)
Comment (biegel) [reply README.md:195]: this should be `GET /wishlists/:Id` above

File: README.md (hunk 6 above)
Comment (biegel) [reply README.md]: 👍

File: README.md (hunk 5 above)
Comment (biegel) [reply README.md]: 👍

**Review by ZiyiHuang96 — APPROVED** [review]

**Pull request merged**